addons/compat_gm/script_component.hpp

```diff
@@ -3,3 +3,6 @@
 
 #include "\z\ace\addons\main\script_mod.hpp"
 #include "\z\ace\addons\main\script_macros.hpp"
+
+#include "\z\ace\addons\csw\script_config_macros_csw.hpp"
+#include "\z\ace\addons\hearing\script_macros_hearingProtection.hpp"
```

Comment:
This causes a requirement on CSW and Hearing.

Reply:
moved to config.cpp

Reply:
How does that fix it? :thinking: 

CSW include should be in a sub-component. But I guess GM doesn't have one? Although I guess those are only needed for building?

Reply:
> How does that fix it? 🤔

Requirement is only there on building.